### PR TITLE
com.apple.mcxprinting

### PR DESCRIPTION
**Pull request by kevinmcox**
Should satisfy #456.

### CODE REVIEW

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist [hunk 1]
```diff
@@ -0,0 +1,439 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+		<key>pfm_description</key>
+		<string>Printing settings</string>
+		<key>pfm_description_reference</key>
+		<string>BLANK</string>
```
Comment (relgit) [inline Manifests/ManifestsApple/com.apple.mcxprinting.plist:8]: Should this stay _BLANK_ or should just be removed?

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+								<key>pfm_type</key>
+								<string>string</string>
+								<key>pfm_value_placeholder</key>
+								<string>file://localhost/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/PrintCore.framework/Resources/Generic.ppd</string>
+							</dict>
+							<dict>
+								<key>pfm_default</key>
+								<false/>
+								<key>pfm_description</key>
+								<string>If true, locks the printer. </string>
```
Comment (relgit) [inline Manifests/ManifestsApple/com.apple.mcxprinting.plist:404]: Space left over after the period.

**Comment by relgit** [comment]
Besides my two nit-picky comments above I'd like to see a documentation URL at the root level too. Other than these it looks good 👍

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist (hunk 1 above)
Comment (kevinmcox) [reply Manifests/ManifestsApple/com.apple.mcxprinting.plist:8]: I added content, thanks!

**Comment by kevinmcox** [comment]
All three changes made, thanks @relgit!

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+		<string>10.7</string>
+		<key>pfm_platforms</key>
+		<array>
+			<string>macOS</string>
+		</array>
+		<key>pfm_subkeys</key>
+		<array>
+			<dict>
+				<key>pfm_default</key>
+				<string>Configures AirPrint settings</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.mcxprinting.plist:31]: Is reference to AirPrint just leftover from originally cloned manifest?

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+				<key>pfm_name</key>
+				<string>PayloadDescription</string>
+				<key>pfm_title</key>
+				<string>Payload Description</string>
+				<key>pfm_type</key>
+				<string>string</string>
+			</dict>
+			<dict>
+				<key>pfm_default</key>
+				<string>AirPrint</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.mcxprinting.plist:45]: Another leftover?

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+				<key>pfm_require</key>
+				<string>always</string>
+				<key>pfm_title</key>
+				<string>Payload Identifier</string>
+				<key>pfm_type</key>
+				<string>string</string>
+			</dict>
+			<dict>
+				<key>pfm_default</key>
+				<string>com.apple.airprint</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.mcxprinting.plist:77]: Another?

**Review by apizz — CHANGES_REQUESTED** [review]
A few very minor fixes and I think we're good to merge

File: Manifests/ManifestsApple/com.apple.mcxprinting.plist (hunk 3 above)
Comment (kevinmcox) [reply Manifests/ManifestsApple/com.apple.mcxprinting.plist]: Ugh, yep, that is exactly what I used as my base to build this one.

**Comment by kevinmcox** [comment]
Thanks @apizz, I fixed those mistakes.

**Review by apizz — APPROVED** [review]

**Pull request merged**